src/Service/EloquaApiClient.php

```diff
@@ -54,14 +54,18 @@ class EloquaApiClient {
    *   LoggerChannelFactoryInterface.
    * @param \Drupal\Core\Cache\CacheBackendInterface $cacheBackend
    *   Cache Backend.
+   * @param \Drupal\Core\Http\ClientFactory $http_client_factory
+   *   A Guzzle client object.
    */
   public function __construct(ConfigFactory $config,
                               LoggerChannelFactoryInterface $loggerFactory,
-                              CacheBackendInterface $cacheBackend) {
+                              CacheBackendInterface $cacheBackend,
+                              ClientFactory $http_client_factory) {
```

Comment:
might want to try to use camel case vars to keep things consistent :)

Reply:
Updated